packages/core/src/server.jsx

```diff
@@ -175,18 +230,22 @@ export default function(defaultStore = appInitialState, headerConfig, reduxMiddl
     <script>
       window.__SSR__ = true;
       window.__APP_NAME__ = "${ req.i18n.options.defaultNS }";
-      window.__HELMET_DEFAULT__ = ${ serialize(headerConfig, {isJSON: true, space: 2}).replace(/\n/g, "\n      ") };
-      window.__INITIAL_STATE__ = ${ serialize(initialState, {isJSON: true, space: 2}).replace(/\n/g, "\n      ") };
+      window.__HELMET_DEFAULT__ = ${serialize(headerConfig, {json: true})};
+      window.__INITIAL_STATE__ = ${serialize(initialState, {json: true})};
```

Comment:
`serialize` only has one parameter

Reply:
Good catch, this was a hold over from swapping `serialize` with `jsesc` in my testing.